compose/buildlocal.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
 set -x -e
 
-ANSIBLE_REPO=galaxyproject/ansible-galaxy-extras
-ANSIBLE_RELEASE=master
+: ${ANSIBLE_REPO:="galaxyproject/ansible-galaxy-extras"}
+: ${ANSIBLE_RELEASE:="master"}
 
-GALAXY_RELEASE=dev
-GALAXY_REPO=galaxyproject/galaxy
+: ${GALAXY_RELEASE:="dev"}
+: ${GALAXY_REPO:="galaxyproject/galaxy"}
 
-docker build --build-arg ANSIBLE_REPO=$ANSIBLE_REPO --build-arg ANSIBLE_RELEASE=$ANSIBLE_RELEASE -t quay.io/bgruening/galaxy-base ./galaxy-base/
-docker build --build-arg GALAXY_REPO=$GALAXY_REPO --build-arg GALAXY_RELEASE=$GALAXY_RELEASE -t quay.io/bgruening/galaxy-init ./galaxy-init/
+: ${TAG_SUFFIX:=""}
```

Comment:
I'm fine with hard coding everything to dev, but this also seems to be fine.
Defaulting to dev probably makes sense to avoid that people who build this locally, without a special tag, will encounter the same problems.

An other idea is to use docker tag and tag these images with different tags, one master and one latest or dev but this is even more confusing I think.